controllers/job.go

```diff
@@ -181,13 +185,34 @@ func (controller *JobController) Post(w http.ResponseWriter, r *http.Request, pa
 		return
 	}
 
-	updateData := struct{ NextState string }{}
+	updateData := struct {
+		NextState string
+		Timeout   time.Duration
```

Comment:
@Bishwajit-Bhattacharjee  Should this be `incrementalTimeout` since it is additive on top of the default system timeout?

Reply:
Sure bhaia, Renaming it.